easybuild/framework/easyconfig/format/one.py

```diff
@@ -136,7 +136,10 @@ def get_config_dict(self):
         cfg_copy = {}
         for key in cfg:
             if key != '__builtins__':
-                cfg_copy[key] = copy.deepcopy(cfg[key])
+                try:
```

Comment:
I think it makes sure sense to check the value type of `cfg[key]` here, and skip the deepcopy for imported modules...

I can't think of anything else that would cause trouble, but if this starts failing because of other changes we make in the future, we don't want to ignore it silently...